src/dbt_score/cli.py

```diff
@@ -94,8 +94,20 @@ def cli() -> None:
     is_flag=False,
     default=None,
 )
+@click.option(
+    "--show",
+    help="Type of output which should be shown"
+    "when using `plain` as `--format`. "
+    "`all` shows all models and all rules"
+    "`failing-models` shows failing rules of failing models"
```

Comment:
A PR has been merged recently to include the linting of sources, could you rename `models` to `items` everywhere? Then it's aligned!